server/memphis_handlers_stations.go

```diff
@@ -659,6 +675,58 @@ func (sh StationsHandler) GetAllStationsDetails(shouldGetTags bool, tenantName s
 	}
 }
 
+func (sh StationsHandler) GetAllStationsDetailsPerTenant(tenantName []string) (map[string]StationsDetailsPerAccount, error) {
```

Comment:
GetAllStationsDetailsPerTenant => GetAllStationsDetailsCrossAllTenants

tenantName => tenantNames

move to cloud